### DISCUSSION TITLE
JWT + Cookie authentication continuously unauthorized

### DISCUSSION THREAD

**Discussion by artemonsh** [fastapi-users/fastapi-users#1001] [Q&A]
I use the [full example](https://fastapi-users.github.io/fastapi-users/10.0/configuration/full-example/) with SQLAlchemy and change only the transport from bearer to cookie and set `cookie_secure` to False. Nevertheless, I keep getting the "Unauthorized" response when trying to get `/users/me` or any other endpoint. I did even set `is_superuser` and `is_verified` to True in the database buy it did not help. Moreover, I don't know why but the cookie is in bytes string format and not a regular string. Can anyone help me with this? 
![image_2022-05-24_18-16-49](https://user-images.githubusercontent.com/53895552/170071498-d4dd0287-01d8-46d4-bd32-9093b557d443.png)

**Comment by frankie567** [comment]
Hi @artemonsh!

I've tried on my side but I can't reproduce the issue. Here is what I have:

```py
cookie_transport = CookieTransport(cookie_max_age=3600, cookie_secure=False)


def get_jwt_strategy() -> JWTStrategy:
    return JWTStrategy(secret=SECRET, lifetime_seconds=3600)


auth_backend = AuthenticationBackend(
    name="jwt",
    transport=cookie_transport,
    get_strategy=get_jwt_strategy,
)
```

Everything works as expected like this.

However, the fact that your cookie value is returned in Python binary format is indeed weird, and probably the cause of the issue. Are you sure you didn't make any other change to the code?
